src/elf.rs

```diff
@@ -8,7 +8,7 @@ const ELFCLASS64: u8 = 2;
 
 // e_type
 /// No file type
-pub const ET_NONE: Elf_Half = 0;
+   pub const ET_NONE: Elf_Half = 0;
```

Comment:
```suggestion
pub const ET_NONE: Elf_Half = 0;
```

Reply:
You can just run `just fmt` on the command line.